libs/ui-lib/lib/common/config/docs_links.ts

```diff
@@ -76,13 +75,11 @@ export const CNV_LINK = 'https://cloud.redhat.com/learn/topics/virtualization/';
 
 export const ODF_LINK = 'https://www.redhat.com/en/resources/openshift-data-foundation-datasheet';
 
-export const getMceDocsLink = (ocpVersion?: string) =>
-  `https://docs.openshift.com/container-platform/${
-    getShortOpenshiftVersion(ocpVersion) || '4.12'
-  }/architecture/mce-overview-ocp.html`;
+export const getMceDocsLink = () =>
```

Comment:
I think this link change is not correct. 
Before, it was to an overview of the MCE operator (similar to ODF_LINK and CNV_LINK), and now it's only for its requirements.

In https://issues.redhat.com/browse/MGMT-18248, it is suggested to use a generic link for all operators, which is 
[link](https://docs.redhat.com/en/documentation/assisted_installer_for_openshift_container_platform/2024/html-single/installing_openshift_container_platform_with_the_assisted_installer/index#customizing-installation_about-ai)

So either use that one (but then CNV and ODF should behave in the same way), or we should have the new link for an overview of the MCE operator.

Reply:
Waiting for a response from QE to see which link we have to use in operators page: the common link or specific link for every operator.

Reply:
We agree to add the specific link for operators. Waiting for the new links to be added here.